perllib/FixMyStreet/Roles/Cobrand/Echo.pm

```diff
@@ -1302,6 +1302,94 @@ sub per_photo_size_limit_for_report_in_bytes {
     return min($max_size_per_image, $max_size_per_image_from_total);
 };
 
+=head2 update_report_for_same_day_amend
+
+This Echo only hook is used to check
+if a Bulky collection that has been changed
+has maintained the same date
+
+=cut
+
+sub update_report_for_same_day_amend {
+        my ($self, $p, $data) = @_;
+
+        my $collection_date = $p->get_extra_field_value('Collection_Date');
```

Comment:
Note that this can be different per cobrand - Kingston/Sutton it is now `Collection_Date_-_Bulky_Items` (might need to also default back to this for old reports though, sorry).